echoregions/tests/test_regions2d.py

```diff
@@ -249,9 +249,9 @@ def test_plot(regions2d_fixture: Regions2D) -> None:
 
 
 @pytest.mark.regions2d
-def test_select_sonar_file(regions2d_fixture: Regions2D, da_Sv_fixture: DataArray) -> None:
+def test_select_sonar(regions2d_fixture: Regions2D, da_Sv_fixture: DataArray) -> None:
     """
-    Tests select sonar file.
+    Tests select_sonar.
```

Comment:
Maybe here these should be `select_sonar_file` to be consistent with the function name?